apropos.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<html>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+
+<head>
+    <link href="https://fonts.cdnfonts.com/css/jsmath-cmbx10" rel="stylesheet">
+    <title>Home - Ma page avec Bootstrap</title>
+    <link rel="stylesheet" href="css/bootstrap.min.css">
+    <link rel="stylesheet" href="css/styles.css">
+</head>
+
+<body>
+    <nav class="navbar font navbar-expand-lg navbar-light bg-light w-100">
```

Comment:
Your menu is not well aligned on mobile. The same for your page content.

![image](https://user-images.githubusercontent.com/6362750/235274666-3d537a0d-6ef7-474d-9fc7-4e99056c078b.png)

![image](https://user-images.githubusercontent.com/6362750/235274695-2ef28851-09b9-4eb7-86ba-6729ce228dd6.png)